CONTRIBUTING.md

```diff
@@ -50,10 +50,14 @@ new engineers.
 2. Get the CockroachDB code:
 
    ```shell
-   git clone https://github.com/cockroachdb/cockroach $(go env GOPATH)/src/github.com/cockroachdb/cockroach
-   cd $(go env GOPATH)/src/github.com/cockroachdb/cockroach
+   cd $(go env GOPATH)/src/github.com/cockroachdb
+   git clone https://github.com/cockroachdb/cockroach
+   cd cockroach
```

Comment:
Wait, no, the old code correctly created `$(go env GOPATH)/src/github.com/cockroachdb/cockroach` if it did not already exist. If you want to go this route to make the positioning within the GOPATH more clear, I think you need to add a `mkdir -p` as well:

```shell
mkdir -p $(go env GOPATH)/src/github.com/cockroachdb
cd $(go env GOPATH)/src/github.com/cockroachdb
git clone https://github.com/cockroachdb/cockroach
cd cockroach
```

Reply:
good call. done